.gitignore

```diff
@@ -1,9 +1,76 @@
-#  Local .terraform directories
-**/.terraform/*
+# Inspired by https://dcmnt.me/2bidtTq
```

Comment:
can we please keep the .gitignore clean. A lot of entries which are not needed for this project